youtube_dl/extractor/cgtn.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import unicode_literals
```

Comment:
You could put a `# coding: utf-8` line at the top in case someone adds a test case with non-ASCII characters. Currently about 1/3 of extractors do this.

Reply:
done!